src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -691,6 +692,7 @@ public static DynamoViewModel Start(StartConfiguration startConfiguration = new
         protected DynamoViewModel(StartConfiguration startConfiguration)
         {
             Dispatcher.CurrentDispatcher.UnhandledException += CurrentDispatcher_UnhandledException;
+            TaskScheduler.UnobservedTaskException += TaskScheduler_UnobservedTaskException;
```

Comment:
So is this a catch-all for all exceptions from any and all Tasks that go unhandled?

Reply:
The documentation says it should catch unobserved exceptions from all tasks.
Not sure what it means to observe a task's exceptions. I think an exception is observed if you use `await task` or `task.Wait()` or `task.Result`.